lib/taskana-core/src/main/java/pro/taskana/monitor/api/reports/item/TimestampQueryItem.java

```diff
@@ -3,7 +3,9 @@
 import pro.taskana.monitor.api.TaskTimestamp;
 import pro.taskana.monitor.api.reports.TimestampReport;
 
-/** The TimestampQueryItem contains the necessary information for the {@link TimestampReport}. */
+/**
+ * The TimestampQueryItem contains the necessary information for the {@linkplain TimestampReport}.
```

Comment:
Isn't the QueryItem like a cell in the report table? If it is, we could specify it this way...

Reply:
It is. A row in the report table consists of QueryItems. Should I change the comment to "The TimestampQueryItem represents the content of a cell in the the {@linkplain TimestampReport}"?